runtime/fail_byt.c

```diff
@@ -34,6 +34,11 @@
 CAMLexport void caml_raise(value v)
 {
   Unlock_exn();
+  if (caml_check_pending_actions()) {
+    CAMLparam1 (v);
+    caml_process_pending_actions();
+    CAMLdrop;
+  }
```

Comment:
```suggestion
  v = caml_process_pending_actions_with_root(v);
```

Reply:
Changed, thanks.

(I added an assertion that `!Is_exception_result(v)`, because failing to call `Extract_exception` before raise is an easy mistake to make, and the implementation of `caml_process_pending_actions_with_root` will do something strange in that case)

Reply:
Sounds good to me.